### DISCUSSION TITLE
Documentation suggestion

### DISCUSSION THREAD

**Discussion by AniOhev** [chrisgurney/obsidian-note-toolbar#50] [General]
I worked out how to add more than one style to a callout, but it might be an idea to explicitly explain this, in the otherwise excellent user documentation

**Comment by chrisgurney** [comment]
@AniOhev Thanks for the feedback! Agree 100% I've added some more details about styling to the [Note Toolbar Callout](https://github.com/chrisgurney/obsidian-note-toolbar/wiki/Note-Toolbar-Callouts#styling) page.
